080-Recipes/010-csv-data/010-import-csv-data-UI.mdx

```diff
@@ -0,0 +1,106 @@
+| Text          | Longer pieces of textual content                          | "Hello world and welcome to Xata" |
+| Integer       | Whole numbers without decimal points                      | `42`                              |
+| Float         | Numbers with decimal points                               | `3.14`                            |
+| Boolean       | Represents true or false values                           | `true`                            |
+| Datetime      | Date and time information                                 | 2023-08-30T00:00:00               |
+| Link to Table | Creates relationships between tables                      | Table: Orders                     |
+| Email         | Email addresses                                           | `user@example.com`                |
+| Multiple      | Multiple values from a predefined set separated by commas | \["Red", "Green"]                 |
+| File          | Reference to an external file, attachment, or document    | `file.pdf`                        |
+
+### Filename formatting
+
+- Ensure that the imported filename/table name is limited to 64 characters or fewer.
+- The table name should consist of alphanumeric characters, hyphens (-), and underscores(\_). Special characters are not allowed.
+
+### Delimiter
+
+A delimiter acts as dividers and distinguish individual data fields within a CSV file. They are useful in structuring the data, allowing for accurate interpretation and parsing.
+
+- The comma (,) is applied as the default delimiter for the CSV import function.
+- The CSV import functionality supports common delimiters such as commas (,), semicolons (;), quotes ( ", ' ), braces ({}), pipes (|), or slashes ( / \ ). You can modify the delimiter in the UI, when importing the CSV file data.
+
+## Import a CSV file using the UI
+
+Xata offers an import tool directly within your dashboard Workspace. This tool enables seamless importing of existing files into the Xata database without causing any downtime. As an alternative, you also have the option to import CSV data using the CLI; however, it's worth noting that the UI method is the recommended approach.
+Importing the CSV file involves transfering data as well as establishing a schema. Once imported, the data in the CSV file will be accessible as a table in your workspace.
+
+1.  Log in to your Xata account and go to the desired Workspace and database. If you haven't created a Workspace as yet, click **Add a Workspace** to do so.
+2.  Choose the preferred branch for your work.
+3.  On the left-hand menu under "Tables," click **Add a table** > **Import a CSV file**.
+4.  Select **Browse Desktop** to choose a supported file to import. Ensure the filename adheres to the formatting recommendations.
+5.  Click **Begin import** to initiate the import.
+
+You'll have acces to partially imported records as the process completes.
+
```

Comment:
Something very important, that is possibly worth noting with like a ⚠️ is that they need to leave the tab open until the import completes.